### DISCUSSION TITLE
Markdown editor won't render email links

### DISCUSSION THREAD

**Discussion by Psichorex** [OrchardCMS/OrchardCore#11589] [Q&A] [answered]
I am working on a bugfix where I have a MarkdownField with Wysiwyg editor and it won't render email links.

`<sample@sample.com> `  --> renders: `<a>sample@sample.com</a>`
`[Contact us](mailto:sample@sample.com)` --> renders: `<a>Contact Us</a>`

I have come to the point where I have 0 clue why is it not working.
Things I know:

- Markdown editor DOES recognize that I'm writing links.
- Writing a link with url e.g. `[Google](https://google.com)` works and renders fine.
- I am using MarkdownField with Wysiwyg editor.
- I am also using MarkdownBodyPart with Wysiwyg editor same problem.

**Comment by Psichorex** [comment, marked as answer]
Solution found:

Setting SanitizeHtml either true or false in the Migration did absolutely nothing.

But when I render the markdown content in .cshtml with `Orchard.MarkdownToHtmlAsync(markdown)`
It seems that when this conversion happens is the time when you need to set sanitizing to FALSE
`Orchard.MarkdownToHtmlAsync(markdown, sanitize: false)` was the solution.

**Reply by Skrypt** [reply]
You should alternatively just configure the Sanitizer to exclude some elements instead of disabling it entirely. This can be done through DI.
